src/style.css

```diff
@@ -1 +1,97 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 700;
+  font-style: normal;
+}
+
+.paige {
```

Comment:
The class name '.paige' seems to be a typo or not meaningful. Ensure class names represent the meaning of the content or the role of the element within your design.